django_project/sawps/views.py

```diff
@@ -140,27 +140,37 @@ def send_invitation_email(self, email_details):
 
 class SendRequestEmail(View):
     def send_email(self, request):
-        """this feature is still being implemented"""
-        subject = 'ORGANISATION REQUEST'
+        organisation = request.POST.get('organisationName')
+        request_message = request.POST.get('message')
+        subject = 'Oganisation Request'
+        if request.user.first_name:
+            if request.user.last_name:
+                name = request.user.first_name + ' ' + request.user.last_name
+            else: name = request.user.first_name
+        else: name = request.user.email
         # Send email
         try:
             message = render_to_string(
-                'emails/invitation_email.html',
+                'emails/request_organisation_email.html',
                 {
                     'domain': Site.objects.get_current().domain,
-                    'email': request.user.email,
+                    'email_address': request.user.email,
+                    'organisation_name': organisation,
+                    'name': name,
+                    'request_message': request_message
                 },
             )
             send_mail(
                 subject,
                 None,
                 settings.SERVER_EMAIL,
-                ['to@gmail.com'],
+                ['amy@kartoza.com'],
```

Comment:
don't put the email directly here. Instead, use the variable from the Django settings 

https://github.com/kartoza/sawps/blob/eb33c7ff255d8f7dc4a974d79f88c2525bf4c6f6/django_project/core/settings/contrib.py#L106